frontend/index.js

```diff
@@ -1,18 +1,19 @@
+import * as animationControls from './tools/animation-controls.js';
 import * as kbd from './tools/keyboard-shortcuts.js';
 import * as menu from './tools/menu.js';
 import * as stats from './tools/stats.js';
 
 import * as environments from './environments.js';
 import * as renderer from './renderer.js';
 import * as radiosity from './radiosity.js';
+import { algorithms } from './algorithms.js';
```

Comment:
Good for having moved the different algorithms to a separate file.
This enhance this code readabilty.

Reply:
Same  thing for the animation controller.